lib/Pipes/Pipes.cpp

```diff
@@ -155,6 +155,9 @@ class DetectABIAnalysis {
 public:
   static constexpr auto Name = "DetectABI";
 
+  static constexpr std::tuple ArgumentInfos = { ArgumentInfo("random_name",
+                                                             0) };
```

Comment:
The `Info` suffix is almost never a good idea. Drop or go for `ArgumentTypes`.
At very least go for `ArgumentsInfo`.